src/flag_gems/utils/triton_lang_extension.py

```diff
@@ -0,0 +1,28 @@
+"""
```

Comment:
Is there any reason why we don't shadow triton.language entirely?

Reply:
I don't want to hack `tl.program_id` or `tl.num_programs` since hacking may introduce some surprises.